recipes/date/all/conanfile.py

```diff
@@ -23,7 +35,51 @@ def source(self):
         for patch in self.conan_data["patches"][self.version]:
             tools.patch(**patch)
 
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["ENABLE_DATE_TESTING"] = False
+        if self.settings.os == "Windows":
+            cmake.definitions["USE_TZ_DB_IN_DOT"] = False
+            cmake.definitions["USE_SYSTEM_TZ_DB"] = False
+        else:
+            cmake.definitions["USE_TZ_DB_IN_DOT"] = self.options.use_tz_db_in_dot
+            cmake.definitions["USE_SYSTEM_TZ_DB"] = self.options.use_system_tz_db
+
+        cmake.definitions["BUILD_SHARED_LIBS"] = self.options.shared
```

Comment:
BUILD_SHARED_LIBS is already set by conan, this line can be removed

Reply:
nice, i thought that this was date's custom define, seems like some kind of convention.

Reply:
applied.